Update cert-notice readme (#2504)
pester/Pester: README.md

> 📦🔐 Pester is now signed. `-SkipPublisherCheck` should no longer be used to install from PowerShell Gallery on Windows 10.

> > 📦🔐 If you upgrade to 5.6.0 you will get a "certificate changed" error if you dont skip publisher check. This is related to a new module certificate. [more info here](#regarding-certificate-for-different-versions).

Reviewer: ```suggestion
> 📦🔐 Upgrading to 5.6.0 you show a "certificate changed" error, this is because a change in Root Certificate, and you have to specify `-SkipPublisherCheck` to update. [More info below.](#560-certificate-error).
```

Reviewer 2: Replace "5.6.0 you" with "5.6.0 will" ?

Not running Windows 10 or facing problems? See the [full installation and update guide](https://pester.dev/docs/introduction/installation).

## Regarding Certificate for different versions

> with 5.6.0 the certificate used for deplyment has changed. this will give you a error when updating.
> below are a list of the used certificates you can expect to be used when importing the module (going back to 2016)

Reviewer: ```suggestion
The certificate used for signing the code has changed in 5.6.0. Error is shown when updating the module.
Below is the list of the certificates you can expect to be used when importing the module (going back to 2016)
```

Version|Authority|Thumbprint
---|---|---
5.6.0++|`CN=DigiCert Trusted G4 Code Signing RSA4096 SHA384 2021 CA1, O="DigiCert, Inc.", C=US`|`2FCC9148EC2C9AB951C6F9654C0D2ED16AF27738`
5.2.0 - 5.5.0|`CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US`|`C7B0582906E5205B8399D92991694A614D0C0B22`
4.10.0 - 5.1.1|`CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US`|`7B9157664392D633EDA2C0248605C1C868EBDE43`
4.4.3 - 4.9.0|`CN=DigiCert SHA2 Assured ID Code Signing CA, OU=www.digicert.com, O=DigiCert Inc, C=US`|`CC1168BAFCDA3B1A5E532DA87E80A4DD69BCAEB1`
3.0.3 - 4.4.2|No Certificate Found|No Certificate Found
3.4.0|`CN=Microsoft Windows Production PCA 2011, O=Microsoft Corporation, L=Redmond, S=Washington, C=US`|`71F53A26BB1625E466727183409A30D03D7923DF`

> in all cases in exeption to version 3.4.0, the Authenticode issuer for certificate should be `CN=Jakub Jareš, O=Jakub Jareš, L=Praha, C=CZ`.

Reviewer: ```suggestion
In all cases, except for version 3.4.0 that was signed directly by Microsoft, the Authenticode issuer for certificate is `CN=Jakub Jareš, O=Jakub Jareš, L=Praha, C=CZ`.
```

> 3.4.0 is a special case as this is the version signed my microsoft and deployed to w10.
> 

Reviewer: ```suggestion
```

> to fix any issue when using a module that have changed certificate is to append `-SkipPublisherCheck` to the install command.

Reviewer: ```suggestion
To successfully update the module when certificate changed, you need to provide `-SkipPublisherCheck` to the `Install-Module` command.
```
